### ISSUE TITLE
Washing machines do not release items when there is no power

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#14222]
(WEB REPORT BY: chazach REMOTE: 172.93.109.202:7777)
Can put items in, can't take them out

Attempting to interact with it will just say
"The washing machine cannot run in this state." with no option to take the clothes out

**Comment by Probe1** [comment]
@Shadowmech88 I smell a kick_act() PR!

**Comment by Intigracy** [comment]
Actually this one would make sense to just fix.

**Comment by Shadowmech88** [comment]
Yeah, there's no reason you shouldn't be able to open the door on an unpowered washing machine.

**Comment by Kitfox88** [comment]
ive seen fancy washing machines that lock the top while they're active so dumbasses can't stuff their hand inside and get it wrenched around by the spin cycles, but im guessing they unlock in case of power failure
